payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardBrandChoiceConfig.kt

```diff
@@ -0,0 +1,11 @@
+package com.stripe.android.ui.core.elements
+
+import com.stripe.android.model.CardBrand
+
+internal sealed interface CardBrandChoiceConfig {
+    data class Eligible(
+        val initialBrand: CardBrand?
```

Comment:
Can this ever be anything but `null`?

Reply:
Yes, if the params of the payment method contain the preferred network, the initial brand will be set to it.